client/js/components/user/DpUserListExtended.vue

```diff
@@ -163,42 +160,40 @@ export default {
       }
 
       ids.map(id => {
+        dplan.notify.notify('confirm', Translator.trans('confirm.user.deleted'))
```

Comment:
Placing the sucess notification outside the `then()` will trigger it also if the operation wasn't successful.

Reply:
done here https://github.com/demos-europe/demosplan-core/pull/2673/commits/8a4018547e97324776689b21b7bdf28226bb7a81